src/harness/fake_sas.py

```diff
@@ -89,7 +89,7 @@ class FakeSas(sas_interface.SasInterface):
   """
 
   def __init__(self):
-    pass
+	pass
```

Comment:
Please undo this change: it's inconsistent with the rest of the file.

Reply:
sorry for all the indentation problems, I think my new IDE (MS visual studio) is treating them automatically and incorrectly :\ .